src/Atc.Test/Atc.Test.csproj

```diff
@@ -16,7 +16,7 @@
   <ItemGroup>
     <PackageReference Include="AutoFixture.AutoNSubstitute" Version="4.17.0" />
     <PackageReference Include="AutoFixture.Xunit2" Version="4.17.0" />
-    <PackageReference Include="FluentAssertions" Version="5.10.3" />
+    <PackageReference Include="FluentAssertions" Version="6.1.0" />
```

Comment:
Breaking change! - I like the ide of upgrading, but be aware of the breaking change from 5.x to 6.x - lot of extensions methods is gone....Please follow up on https://github.com/fluentassertions/fluentassertions/blob/master/docs/_pages/releases.md#600 -> BreakingChanges.

If we upgrade this package, please make sure that all ATC.xxxx projects are aligned / upgraded on this part (version 6.1).

Reply:
Sorry for moving a bit too fast on this PR. I think you have a good point, and the upgrade of FluentAssertions were done a bit automatic and was not really required by the bug fix.

However, I do think we need to follow the tool updates to make sure we do not lack too far behind. 

I would suggest that the effort of keeping everything aligned should be reconsidered, so we do not update atc.xxx libraries just because the library we use for testing has a new version - but instead wait until each library has another need to change.

This way the task becomes more manageable, and we do not get a ton of updates to the atc.xx libraries that brings no value.

Reply:
I agree with @rickykaare, it should be possible for each library to control it's own dependency upgrades. As you pointed out @davidkallesen this is a breaking change that we might be willing to take here, but definitely don't need to do for all. Imposing dependency alignment for all packages will kill progress and serves no real purpose.